doi-transformer/src/test/java/no/unit/nva/doi/transformer/CrossRefConverterTest.java

```diff
@@ -134,9 +186,10 @@ public void entityDescriptionContainsTheEarliestYearFoundInPublishedPrintField()
 
     @Test
     @DisplayName("toPublication sets null EntityDescription date when input has no \"issued\" date")
-    public void entityDescriptionDateIsNullWhenInputDataHasNoPublicationDate() throws InvalidIssnException {
-        sampleInputDocument.setIssued(null);
-        Publication publicationWithoutDate = toPublication(sampleInputDocument);
+    public void entityDescriptionDateIsNullWhenInputDataHasNoPublicationDate()
```

Comment:
it is hard to see that the display name and the method name are describing the same thing

Reply:
Name and and description somewhat closer to each other...